src/design-system/Modal.tsx

```diff
@@ -0,0 +1,43 @@
+import React, { useState } from "react";
+import styled from "styled-components";
+
+import Colors from "./Colors";
+import ModalDialog from "./ModalDialog";
+import { TitleProps } from "./ModalTitle";
+
+const ModalContainer = styled.div``;
+
+const ModalTrigger = styled.button`
+  color: ${Colors.green};
+  cursor: pointer;
+  font-family: 'Libre Baskerville', serif;
+  font-size: 32px;
+  line-height: 32px
+  letter-spacing: -0.03em
+`;
+
+interface Props {
+  modalTitle?: TitleProps["title"];
```

Comment:
does `TitleProps.title` work here?

Reply:
It doesn't, I originally tried that and the linter suggested using `TitleProps["title"]`